pyproject.toml

```diff
@@ -14,9 +14,11 @@ readme = "README.md"
 requires-python = ">=3.8"
 license = {text = "BSD"}
 dependencies = [
+        "kiwisolver",
         "PyIBIS-AMI>=4.1",
         "pyside2",
         "pyyaml>=6",
+        "qt>=5",
```

Comment:
Pypi doesn't have a built binary package `qt` so this will cause pip installs to fail.  I believe `pyside2` installs qt for us at least in pypi/pip world.  If you need the explicit qt for conda, we will have to move it back to the meta.yaml files.

Reply:
Nice catch!
I just pushed a fix, following your guidance.